common/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     implementation("com.fasterxml.jackson.datatype:jackson-datatype-jdk8:$jackson_version")
     implementation("io.micrometer:micrometer-core")
     implementation('org.hibernate:hibernate-validator:8.0.1.Final')
+    implementation("com.github.ben-manes.caffeine:caffeine:3.2.0")
```

Comment:
lets use the lib manifest way of declaring the dependency.

Check:
https://github.com/flipkart-incubator/varadhi/blob/83c829a91de673379e43613bdb6fa36af0e96981/server/build.gradle#L21
&
https://github.com/flipkart-incubator/varadhi/blob/83c829a91de673379e43613bdb6fa36af0e96981/gradle/libs.versions.toml#L13